src/plugins/workspace/server/saved_objects/workspace_id_consumer_wrapper.ts

```diff
@@ -64,7 +141,14 @@ export class WorkspaceIdConsumerWrapper {
         ),
       delete: wrapperOptions.client.delete,
       find: (options: SavedObjectsFindOptions) =>
-        wrapperOptions.client.find(this.formatWorkspaceIdParams(wrapperOptions.request, options)),
+        wrapperOptions.client.find(
+          this.formatWorkspaceIdParams(
+            wrapperOptions.request,
+            // The `formatFindParams` is a workaroud for 2.14 to always list global data sources,
```

Comment:
`workaroud` -> `workaround`

Reply:
Thanks for catching this, done for that.